dae-alright/src/components/Modal/CustomModal/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Modal from "react-bootstrap/Modal";
+import { Button } from "react-bootstrap";
+import "./index.scss";
+
+const CustomModal = props => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
```

Comment:
Why do we still have this? We agreed to allow the parent component that will be using this to handle the open and close.

Reply:
I have tried to do that but I got an error, these are the things that control the modal,